services/app-web/src/components/ContractTable/ContractTable.tsx

```diff
@@ -431,6 +430,26 @@ export const ContractTable = ({
         setColumnFilters([])
     }
 
+    useEffect(() => {
+        // if on root route
+        if (location.hash === '' && showFilters) {
+            updateFilters(
+                statusColumn,
+                [
+                    {
+                        label: 'Submitted',
+                        value: 'SUBMITTED',
+                    },
+                    {
+                        label: 'Unlocked',
+                        value: 'UNLOCKED',
+                    },
+                ],
+                'status'
+            )
+        }
+    })
```

Comment:
```suggestion
    }, [])
```
nit: I think we only want to run this useEffect once on initial render? If so we could give it a empty dependency array so it only tuns once, otherwise anything that causes a re-render, like selecting filters, will cause this useEffect to fire. The conditional would skip `updatingFilters` but we could avoid firing the useEffect entirely.

Reply:
@JasonLin0991 made this change but added the dependencies, otherwise an empty array kicks up a linting error

Reply:
That's good with me. 

The other way is to add `// eslint-disable-next-line react-hooks/exhaustive-deps`.